react/components/FollowButton/index.js

```diff
@@ -66,12 +66,14 @@ class FollowButton extends Component {
       ...rest
     } = this.props;
 
+    const is_followed = !!(followable && followable.is_followed);
```

Comment:
What's this aboot?

Reply:
Forcing a boolean when the value doesn't exist. I will remove this, I just wanted to develop on Storybook first.

Reply:
Ah weird yeah; it should never be nil. But I dunno what ur doing in Storybook